docs/guide/elasticsearch-dsl.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[elasticsearch-dsl]]
+== Elasticsearch DSL
```

Comment:
I would call this the "Elasticsearch Python DSL" to clarify that it is not a general-purpose DSL. Now that it is documented next to the Elasticsearch docs, this seems important.